### PR TITLE
1.1.2 release 

### PR DESCRIPTION
**Pull request by jfy133**
Bug fixes release 1.1.2!

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]

File: subworkflows/local/standardisation_profiles.nf [hunk 1]
```diff
@@ -52,12 +52,19 @@ workflow STANDARDISATION_PROFILES {
...
+                                        meta_new.db_name = meta.db_name
                                         [meta_new, profile]
                             }
                             .groupTuple ()
-                            .map { [ it[0], it[1].flatten() ] }
+                            .map {
+                                meta, profiles ->
+                                    meta = meta + [
+                                        tool: meta.tool == 'kraken2-bracken' ? 'kraken2' : meta.tool, // replace to get the right output-format description
+                                        id: meta.tool == 'kraken2-bracken' ? "${meta.db_name}-bracken" : "${meta.db_name}" // append so to disambiguate when we have same databases for kraken2 step of bracken, with normal bracken
```
Comment (sofstam) [inline subworkflows/local/standardisation_profiles.nf:64]: More of a question but is the id referring to the sample?

Comment (jfy133) [reply subworkflows/local/standardisation_profiles.nf:64]: At this point no, as we want to aggregate by tool/db, as it's the taxoasta merging into one table of multiple samples

Comment (jfy133) [reply subworkflows/local/standardisation_profiles.nf:64]: At this point no, as we want to aggregate by tool/db, as it's the taxoasta merging into one table of multiple samples

Comment (sofstam) [reply subworkflows/local/standardisation_profiles.nf:64]: It is the id for distinguishing the `tool` from the `db`, then?

Comment (jfy133) [reply subworkflows/local/standardisation_profiles.nf:64]: yes :)

Comment (sofstam) [reply subworkflows/local/standardisation_profiles.nf:64]: Cool, the PR looks good to me, great job 🚀

**Review by sofstam — APPROVED** [review]
🚀

**Pull request merged**